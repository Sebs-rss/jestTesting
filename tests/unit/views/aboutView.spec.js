// AquÃ­ creo aqui las pruebas unitarias para el componente AboutView.vue

/* import { mount } from '@vue/test-utils'*/
import { shallowMount } from '@vue/test-utils'
import AboutView from '@/views/AboutView.vue'

/* describe('AboutView.vue', () => {
    test('Probando la existencia del componente o vista AboutView ', async () => {
        const wrapper = mount(AboutView)
        expect(wrapper.findComponent(AboutView).exists()).toBe(true)
    })
}) */

//Con shallowMount
describe('AboutView.vue', () => {
    test('Probando la existencia del componente o vista AboutView ', async () => {
        const wrapper = shallowMount(AboutView)
        expect(wrapper.findComponent(AboutView).exists()).toBe(true)
    }),
    it('matches the snapshot', () => { const wrapper = shallowMount(AboutView); expect(wrapper.html()).toMatchSnapshot(); });
})