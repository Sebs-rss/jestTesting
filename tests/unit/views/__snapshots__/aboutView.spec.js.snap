// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView.vue matches the snapshot 1`] = `""`;
